Apps/VisionOS/VisionOS/AppDelegate.swift

```diff
@@ -30,6 +30,15 @@ class AppDelegate: NSObject, UIApplicationDelegate {
                 config.logLevel = .debug
             }
         }
+        
+        // If user already logged in, let's
+        // call identify
+        let profile = AppState.shared.profile
+        if profile.loggedIn {
+            CustomerIO.shared.identify(
```

Comment:
Wondering why is `identified` being called again? 🤔 
Our SDK retains the last logged in user id. Do you have a use case you want to cover with this ?

Reply:
Good call out, I didn't know that!
Will remove this call